apiv2/internal/server/host.go

```diff
@@ -677,38 +677,25 @@ func (is *InventorygRPCServer) RegisterUpdateHost(
 	return invHost, nil
 }
 
-func (is *InventorygRPCServer) listAllHosts(ctx context.Context, filter string) ([]*computev1.HostResource, error) {
-	var offset int
-	var pageSize int32 = 100
-	hasNext := true
-	hosts := make([]*computev1.HostResource, 0, pageSize)
+func (is *InventorygRPCServer) listHosts(ctx context.Context, filter string) (int32, error) {
```

Comment:
the name `listHosts` could be misleading. We are actually returning only the total number
```suggestion
func (is *InventorygRPCServer) totalHosts(ctx context.Context, filter string) (int32, error) {
```

Reply:
Done.